eth/tracers/api_blocktrace.go

```diff
@@ -109,5 +110,18 @@ func (api *API) createTraceEnvAndGetBlockTrace(ctx context.Context, config *Trac
 	}
 
 	chaindb := api.backend.ChainDb()
-	return api.scrollTracerWrapper.CreateTraceEnvAndGetBlockTrace(api.backend.ChainConfig(), api.chainContext(ctx), api.backend.Engine(), chaindb, statedb, parent, block, true)
+	// create a copy of api.backend.ChainConfig to modify
+	chainConfig := new(params.ChainConfig)
+	*chainConfig = *api.backend.ChainConfig()
+	if config != nil && config.Overrides != nil {
+		// the merge.Merge seems not work well
+		// mergo.Merge(chainConfig, config.Overrides, mergo.WithOverride)
```

Comment:
```suggestion
		// merge.Merge(chainConfig, config.Overrides, mergo.WithOverride)
```

Reply:
the lib is called `mergo` (`dario.cat/mergo`)

Reply:
got it.